Hazelcast.Test/Hazelcast.Test.csproj

```diff
@@ -29,4 +29,8 @@
   <ItemGroup>
     <EmbeddedResource Include="Resources\*" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
We don't need these as above `<EmbeddedResource Include="Resources\*" />` adds everthing

Reply:
Fixed